wxt_cidrbot/git_webhook_handler.py

```diff
@@ -199,12 +223,83 @@ def triage_issue(self, installation_id, json_string, x_event_type):
                 self.logging.debug("assigning result %s", reply_message)
                 if 'Error: **invalid user**' in reply_message:
                     self.logging.debug("Invalid user, cannot assign %s: ", user_to_assign)
-                    #self.dynamo.remove_triage_user(user_to_assign, room_id)
                     fail_triage_message = f"{user_to_assign} cannot be assigned because they do not have access to the repo/org, checking next user in triage list..."
                     self.Api.messages.create(room_id, markdown=fail_triage_message, parentId=msg_edit_id)
                     continue
                 break
-        self.Api.messages.create(room_id, markdown=reply_message, parentId=msg_edit_id)
+
+        room_message = f"{hyperlink_format} successfully assigned to " + user_to_assign
+        if isinstance(reply_message, list):
+            if reply_message[1] is not None and reply_message[1] == 'notify user':
+                room_message = reply_message[3]
+
+        self.Api.messages.create(room_id, markdown=room_message, parentId=msg_edit_id)
+
+    def send_codeowners_message(
+        self, issue, room_id, hyperlink_format, hyperlink_format_repo, issue_type, installation_id, json_string
+    ):
+        issue = issue.as_pull_request()
+        reviewers = issue.raw_data['requested_reviewers']
+
+        self.logging.debug("Codeowners is active, sending update message and quitting")
+        reviewer_message = ""
+        for reviewer in reviewers:
+            reviewer_message += reviewer['login'] + ', '
+        reviewer_message = reviewer_message[:-2]
+        empty_triage_message = f"{issue_type} {hyperlink_format} created in {hyperlink_format_repo}. This {issue_type} is auto assigned to {reviewer_message} via codeowners"
+        self.Api.messages.create(room_id, markdown=empty_triage_message)
+        self.logging.debug("Sending message to code owners")
+        self.send_review_message(installation_id, json_string, True, reviewers)
+        sys.exit(1)
```

Comment:
Generally you probably don't want to use sys.exit() at the end of random function.  It's probably ok in this case, but what if I was using this as a library?  It would unexpected just bail the whole script and maybe I had more stuff to do.

Reply:
Resolved to use return statements in place of all sys exits